node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -161,6 +163,68 @@ impl<I> Era<I> {
     pub(crate) fn set_paused(&mut self, paused: bool) {
         self.consensus.set_paused(paused);
     }
+
+    /// Computes rewards and returns the era report.
+    pub(crate) fn create_report(
```

Comment:
From the code structuring perspective - it looks like the rewards are now era-independent which means they're also protocol-independent? So we get the same rewards regardless of what type of protocol is running in that era?

Reply:
Yes, exactly.
The protocol only has to report faults (equivocations) and inactive validators; apart from that it doesn't influence the rewards anymore.